### PR TITLE
[DOC] Clarification about inhertiance.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/contracts/inheritance.rst [hunk 1]
```diff
@@ -6,13 +6,19 @@ Inheritance
...
+Polymorphism means that a function call (internal and external)
+always executes the function of the same name (and parameter types)
+in the most derived contract in the inheritance hierarchy.
+This has to be explicitly enabled on each function in the
+hierarchy using the ``virtual`` and ``override`` keywords.
 See :ref:`Function Overriding <function-overriding>` for more details.
 
+It is possible to call functions further up in the inheritance
+hierarchy internally by explicitly specifying the contract
+using ``ContractName.functionName()`` or using ``super.functionName()``
```
Comment (ekpyron) [inline docs/contracts/inheritance.rst:18]: Should we mention that one may need to change them from ``external`` to ``public`` for this here or in the "How to update your code" part? Or we can add the ``= Base.superFunction;`` later, once we get around doing that...

Comment (chriseth) [reply docs/contracts/inheritance.rst:18]: "possible to call ... internally ..." - that's of course not possible if the function cannot be called internally at all. This sentence here is unrelated to any 0.6.0 change.

File: docs/060-breaking-changes.rst [hunk 2]
```diff
@@ -64,7 +64,7 @@ This section highlights changes that affect syntax and semantics.
   which send value will revert. If you only implement the receive and not the fallback function, calling a non-existing function on your contract will now revert. You should only need to implement the fallback function
   if you are following an upgrade or proxy pattern.
 
-* Functions can now only be overridden when they are either marked with the ``virtual`` keyword or defined in an interface. When overriding a function or modifier, the new keyword ``override`` must be used. When overriding a function or modifier defined in multiple parallel bases, all bases must be listed in parentheses after the keyword like so: ``override(Base1, Base2)``.
+* Functions can now only be overridden when they are either marked with the ``virtual`` keyword or defined in an interface. Functions without implementation outside of an interface have to be marked ``virtual``. When overriding a function or modifier, the new keyword ``override`` must be used. When overriding a function or modifier defined in multiple parallel bases, all bases must be listed in parentheses after the keyword like so: ``override(Base1, Base2)``.
```
Comment (leonardoalt) [inline docs/060-breaking-changes.rst]:
```suggestion
* Functions can now only be overridden when they are either marked with the ``virtual`` keyword or defined in an interface. Functions without implementation outside an interface have to be marked ``virtual``. When overriding a function or modifier, the new keyword ``override`` must be used. When overriding a function or modifier defined in multiple parallel bases, all bases must be listed in parentheses after the keyword like so: ``override(Base1, Base2)``.
```

Comment (leonardoalt) [reply docs/060-breaking-changes.rst]: I think `of` is not needed in such cases

File: docs/060-breaking-changes.rst [hunk 3]
```diff
@@ -90,7 +90,7 @@ This section gives detailed instructions on how to update prior code for every b
 
 * Choose unique identifiers for variable declarations in inline assembly that do not conflict with declartions outside the inline assembly block.
 
-* Add ``virtual`` to every non-interface function you intend to override.  For single inheritance, add ``override`` to every overriding function. For multiple inheritance, add ``override(A, B, ..)``, where you list all contracts that define the overridden function in the brackets. When multiple bases define the same function, the inheriting contract must override all conflicting functions.
+* Add ``virtual`` to every non-interface function you intend to override. Add ``virtual`` to all functions without implementation outside of interfaces. For single inheritance, add ``override`` to every overriding function. For multiple inheritance, add ``override(A, B, ..)``, where you list all contracts that define the overridden function in the brackets. When multiple bases define the same function, the inheriting contract must override all conflicting functions.
```
Comment (leonardoalt) [inline docs/060-breaking-changes.rst]: Same here.

File: docs/060-breaking-changes.rst (hunk 2 above)
Comment (leonardoalt) [reply docs/060-breaking-changes.rst]: @ChrisChinchilla ?

**Pull request merged**